tini/src/seccomp_fd_notify.c

```diff
@@ -150,21 +150,6 @@ struct sock_filter net_perf_filter[] = {
 		/* Trap connect */
 		BPF_JUMP(BPF_JMP | BPF_JEQ | BPF_K, __NR_connect, 0, 1),
 		BPF_STMT(BPF_RET + BPF_K, SECCOMP_RET_USER_NOTIF),
-		/* Trap perf-related syscalls */
```

Comment:
This is ok in the short term, but eventually we are going to want perf calls on things handling `net` too.

Reply:
Agreed. Probably cleanest with the clone() mechanism to send the notify fd.